assets/javascripts/modules/ajaxCallbacks.js

```diff
@@ -107,6 +107,94 @@ var ajaxCallbacks = {
       }
     }
   },
+  apiCollaboratorResponse: {
+    callbacks: {
+      success: function(response, $element, data, helpers, targets, container, type) {
+        var email = $element.find('[name=email]').val();
+        var list = $('[data-collaborator-list]');
+        var row = list.children().first().clone(true);
+        var children = row.children();
+        var form = children.last().find('.form');
+        var message = response.registeredUser ?
+                    'We have sent an email to <strong>'+ email +'</strong> confirming they have been added to this application.' :
+                    'We have sent an email to <strong>'+ email +'</strong> inviting them to register with the API Developer Hub. ' +
+                    'They cannot access the application until they register.';
```

Comment:
`'We have sent an email to <strong>'+ email +'</strong>` could be in a variable?